sphinx_wagtail_theme/layout.html

```diff
@@ -36,6 +36,13 @@
     {%- endif %}
     <link rel="stylesheet" type="text/css" href="{{ pathto('_static/' + style, 1) }}" />
     <link rel="stylesheet" type="text/css" href="{{ pathto('_static/dist/fontawesome.css', 1) }}" />
+    {%- for css in css_files %}
```

Comment:
Should this be `html_css_files` per L98 of `docs/customizing.rst`, or am I misunderstanding something about how this works?

```suggestion
    {%- for css in html_css_files %}
```

Reply:
Nope, it is available in the jinja context as `css_files`. Then generally follow that naming pattern (e.g. html_theme_options becomes theme_options). See basic template which is where this code was ripped from: https://github.com/sphinx-doc/sphinx/blob/634dd395cca6c4eae2762bfbb2b6a8b1f109df00/sphinx/themes/basic/layout.html#L95-L101

Reply:
How fascinating and weird 😆

Reply:
Yea sphinx internals are a bit grisly in places. I'm sure at some point I will end up refactoring the whole layout template to inherit from basic, until then it's copy-pasta for dinner.